content/en/blog/_posts/2022-12-08-kubernetes-1.26-blog.md

```diff
@@ -0,0 +1,355 @@
+* [Azure file in-tree to CSI driver migration](https://github.com/kubernetes/enhancements/issues/1885)
+* [Job tracking without lingering Pods](https://github.com/kubernetes/enhancements/issues/2307)
+* [Service Internal Traffic Policy](https://github.com/kubernetes/enhancements/issues/2086)
+* [Kubelet Credential Provider](https://github.com/kubernetes/enhancements/issues/2133)
+* [Support of mixed protocols in Services with type=LoadBalancer](https://github.com/kubernetes/enhancements/issues/1435)
+* [Reserve Service IP Ranges For Dynamic and Static IP Allocation](https://github.com/kubernetes/enhancements/issues/3070)
+* [CPUManager](https://github.com/kubernetes/enhancements/issues/3570)
+* [DeviceManager](https://github.com/kubernetes/enhancements/issues/3573)
+
+### Deprecations and removals
+
+12 features were [deprecated or removed](/blog/2022/11/18/upcoming-changes-in-kubernetes-1-26/) from
+Kubernetes with this release.
+    
+* [CRI `v1alpha2` API is removed](https://github.com/kubernetes/kubernetes/pull/110618)
+* [Removal of the `v1beta1` flow control API group](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#flowcontrol-resources-v126)
+* [Removal of the `v2beta2` HorizontalPodAutoscaler API](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#horizontalpodautoscaler-v126)
+* [GlusterFS plugin removed from available in-tree drivers](https://github.com/kubernetes/enhancements/issues/3446)
+* [Removal of legacy command line arguments relating to logging](https://github.com/kubernetes/kubernetes/pull/112120)
+* [Removal of `kube-proxy` userspace modes](https://github.com/kubernetes/kubernetes/pull/112133)
+* [Removal of in-tree credential management code](https://github.com/kubernetes/kubernetes/pull/112341)
+* [The in-tree OpenStack cloud provider is removed](https://github.com/kubernetes/enhancements/issues/1489)
+* [Removal of dynamic kubelet configuration](https://github.com/kubernetes/kubernetes/pull/112643)
+* [Deprecation of non-inclusive `kubectl` flag](https://github.com/kubernetes/kubernetes/pull/113116)
+* [Deprecations for `kube-apiserver` command line arguments](https://github.com/kubernetes/kubernetes/pull/38186)
+* [Deprecations for `kubectl run` command line arguments](https://github.com/kubernetes/kubernetes/pull/112261)
+
+### Release notes
+
+The complete details of the Kubernetes v1.26 release are available in our [release
+notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.26.md).
+
+### Availability
+
+Kubernetes v1.26 is available for download on
+[GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.26.0). To get started with
+Kubernetes, check out these [interactive tutorials](/docs/tutorials/) or run local Kubernetes
+clusters using containers as "nodes", with [kind](https://kind.sigs.k8s.io/). You can also easily
+install v1.26 using [kubeadm](/docs/setup/independent/create-cluster-kubeadm/).
```

Comment:
_Something for the release retro:_
these days, we could link to https://k8s.io/releases/download/

Reply:
I can make that change if that's better, I used the same formula as previous releases but it might be time to update them.

Reply:
Your call @fsmunoz - I don't mind if _this_ release uses the old way.

Reply:
https://k8s.io/releases/download/ has details about how to validate that what you download has been signed.

Reply:
@kubernetes/release-managers does one might now more here?

Reply:
The [preview](https://deploy-preview-37810--kubernetes-io-main-staging.netlify.app/releases/download/) of https://k8s.io/releases/download/ looks broadly right although some details are stale AFAICT (eg: artifact signing is going to be beta)

Reply:
Given the instructions on the artefact signing, which is an important theme in the release article itself, it might make sense to change it. Will push a commit with it, hadn't thought about it in that light.

Reply:
Cool, sounds good.